test/utils/readRowsServiceParameters.ts

```diff
@@ -22,8 +22,8 @@ interface SharedReadRowsParameters {
 }
 
```

Comment:
`ReadRowsServiceParameters` interface is generalized so that `keyFrom` and `keyTo` do not have to be provided. When they are not provided, the parameters from the request are used.